rust/src/helpers/array.rs

```diff
@@ -23,14 +23,28 @@ where
     }
 }
 
+#[allow(clippy::range_plus_one, clippy::too_many_lines)]
 pub fn sort<F>(node: &SyntaxNode, transform: F)
 where
     F: Fn(&str) -> String,
 {
     for array in node.children_with_tokens() {
         if array.kind() == ARRAY {
             let array_node = array.as_node().unwrap();
-            let mut value_set = Vec::<Vec<SyntaxElement>>::new();
+            let has_trailing_comma = array_node
+                .children_with_tokens()
+                .map(|x| x.kind())
+                .filter(|x| *x == COMMA || *x == VALUE)
+                .last()
+                == Some(COMMA);
+            let mut value_set: Vec<
+                Vec<
+                    taplo::rowan::NodeOrToken<
```

Comment:
Let's not use namespaces and instead import the class directly, it's how we do everywhere else to keep this more readable.

Reply:
(I fixed this already, I was debugging and injected the full type to understand something)